services/blockstorage/test/harness.go

```diff
@@ -55,18 +81,51 @@ func (d *driver) failNextBlocks() {
 	d.storageAdapter.FailNextBlocks()
 }
 
-func NewDriver() *driver {
```

Comment:
Why not rename `Driver` to `Harness` to match the rest of the code base? In the acceptance and other services it's always named harness

Reply:
Done.